mod/activity/views/default/river/filter.mjs

```diff
@@ -0,0 +1,15 @@
+/**
+ * Initiates page reload when river selector value changes
+ * @module core/river/filter
+ */
+
+import $ from '../jquery.mjs';
```

Comment:
With jQuery we probably _could_ just import the existing `.js` file, but it would double-load the file and possibly return a different object ref because ES modules have no idea what's been loaded by require().